src/doc/trpl/no-stdlib.md

```diff
@@ -71,18 +73,18 @@ The compiler currently makes a few assumptions about symbols which are available
 in the executable to call. Normally these functions are provided by the standard
 library, but without it you must define your own.
 
-The first of these three functions, `stack_exhausted`, is invoked whenever stack
+The first of these functions, `stack_exhausted`, is invoked whenever stack
 overflow is detected.  This function has a number of restrictions about how it
 can be called and what it must do, but if the stack limit register is not being
 maintained then a thread always has an "infinite stack" and this function
 shouldn't get triggered.
 
-The second of these three functions, `eh_personality`, is used by the
-failure mechanisms of the compiler. This is often mapped to GCC's
-personality function (see the
-[libstd implementation](../std/rt/unwind/index.html) for more
+The second and third of these functions, `eh_personality` and `rust_eh_unwind_resume`,
+are used by the failure mechanisms of the compiler. These are often mapped to GCC's
+personality function and libgcc's `_Unwind_Resume` respectively
```

Comment:
Wording of this sentence somehow seems to suggest GCC personality function and `_Unwind_Resume` come from two different locations, while in practice they both come from `libgcc`.